src/core/plugins/core-commands/server/commands/moderator/inventory.ts

```diff
@@ -41,19 +41,26 @@ class InventoryCommands {
 
     @command('getitem', LocaleController.get(LOCALE_KEYS.COMMAND_GET_ITEM, '/getitem'), PERMISSIONS.ADMIN)
     private static async handleGetItem(player: alt.Player, ...args) {
+        const amount = args[args.length - 1];
+
+        args.pop();
```

Comment:
should only pop if the last argument is a number and should verify length first
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN

if not present, default the amount to 1